app/src/main/java/com/framgia/forder/screen/cart/ItemShoppingCartViewModel.java

```diff
@@ -74,4 +82,17 @@ public String getShopName() {
     public String getTotal() {
         return mCart.getFormatTotal();
     }
+
+    public String getNumberOfProduct() {
+        return Integer.toString(mCart.getNumberOfProduct());
+    }
+
+    public String getOrderTime() {
+        return mCartItem.getStartHour() + "-" + mCartItem.getEndHour();
+    }
+
+    public boolean getIsProductTimeOut() {
```

Comment:
WARNING: [BooleanGetMethodName] A getX() method which returns a boolean should be named isX()
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/naming.html#BooleanGetMethodName